solutions/day-17/level2/src/components/NavigationBar.js

```diff
@@ -0,0 +1,26 @@
+import { NavLink } from "react-router-dom";
+
+import "../styles/NavigationBar.css";
+
+const NavigationItem = ({ path, title }) => (
+  <NavLink
+    to={path}
+    className={({ isActive }) =>
+      isActive ? "navigation-item-selected" : "navigation-item"
+    }
+  >
+    {title}
+  </NavLink>
+);
+
+const NavigationBar = () => {
+  return (
+    <div className="navigation-bar">
+      <NavigationItem path={"/"} title="Home" />
+      <NavigationItem path={"experiences"} title="Experiences" />
+      <NavigationItem path={"contact"} title="Contact" />
```

Comment:
replace the braces with quotes since you're just passing a string.

Reply:
Updated. Thanks.